src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.ts

```diff
@@ -33,4 +75,134 @@ export class DaoStageContent {
     this.data.social_medias.splice(index, 1);
     this.form.revalidateErrors();
   }
+
+  async resolveENS(name: string): Promise<string> {
```

Comment:
Why can't this be used
https://github.com/PrimeDAO/prime-deals-dapp/blob/397b22b66feeac6512b9752dc531719546469165/src/services/EthereumService.ts#L350
or the `EnsService`?